README.md

```diff
@@ -31,7 +31,7 @@ npm install
 # Copy the webpack configuration from webpack.config.sample.js to webpack.config.js
 cp webpack.config.sample.js webpack.config.js
 
-# Locate appdata\owa folder and copy the path
+# Locate the 'appdata\owa' directory in the reference application folder and type the following command to get the path to the 'appdata\owa' directory and copy it to the clipboard
```

Comment:
@dkayiwa an example of the present working directory is /Users/andeladeveloper/Downloads/referenceapplication-standalone-2.6.0/appdata\owa